### DISCUSSION TITLE
headless mode advice

### DISCUSSION THREAD

**Discussion by kvvvotheOnnen** [seleniumbase/SeleniumBase#3214] [General]
hi, all my test and scrap wil end in a only terminal linux, is there something like general headless-mode advice for UC mode?

**Comment by mdmintz** [comment]
Headless Mode isn't needed for Linux in UC Mode because there's a special Xvfb virtual display.

**Reply by kvvvotheOnnen** [reply]
UC Mode will stay undetectable if i'm only using with SB(uc=True, headless=True, xvfb=True) as sb?

**Reply by mdmintz** [reply]
Don't use Headless Mode with UC Mode.
